assembl/views/api2/discussion.py

```diff
@@ -743,9 +752,25 @@ def get_visitors(request):
     for st in discussion.agent_status_in_discussion:
         if not getattr(st, attribute, None):
             continue
+        profile_fields = db.query(m.AbstractConfigurableField, m.ProfileField).filter(m.ProfileField.discussion_id == discussion.id).join(m.ProfileField.configurable_field).filter(
+            m.ProfileField.agent_profile_id == st.agent_profile.id).filter(m.AbstractConfigurableField.identifier == "CUSTOM").filter(m.ProfileField.configurable_field_id == m.AbstractConfigurableField.id).all()
```

Comment:
You can use `&&` operator in the first `filter` method and chain your filters that way instead of calling `filter` multiple times. Both works.

Reply:
Also, don't manually run `CUSTOM` here. The check should be against `ConfigurableFieldIdentifiersEnum.CUSTOM.value`